docs/connectors/taps.rst

```diff
@@ -38,6 +36,27 @@ PipelineWise can replicate data from the following data sources:
 
         :ref:`tap-postgres`
 
+    .. container:: tile
+
+        .. container:: img-hover-zoom
+
+          .. image:: ../img/mongodb-logo.png
+             :target: taps/mongodb.html
+
+        :ref:`tap-mongodb`
+
+
+.. container:: tile-wrapper
+
+    .. container:: tile
+
+        .. container:: img-hover-zoom
+
+          .. image:: ../img/snowflake-logo.png
+             :target: taps/snowflake.html
+
+        :ref:`tap-snowflake`
+
     .. container:: tile
```

Comment:
4 columns layout not rendered correctly on phones and page looks broken from small displays :(
that was a problem in the very beginning

Reply:
Oh! that's a fair point, let me look it up.

Reply:
I reverted back to 2 tiles per line. Did some light search, couldn't find responsive themes.